cfme/services/requests.py

```diff
@@ -1,286 +1,370 @@
+                logger.debug(' Request Message: %s', row.last_message.text)
+                return row
+        else:
+            raise Exception("The requst specified by {} not found!".format(str(cells)))
+            return False
+
+    @property
+    def in_requests(self):
+        return (self.logged_in_as_current_user and
+                # TODO uncomment after BZ 1472844 is fixed
+                # self.navigation.currently_selected == ['Services', 'Requests'] and
+                self.title.text == 'Requests')
+
+    def open_request(self, cells):
+        row = self.table.row(description=cells)
+        row.description.click()
+
+    @property
+    def is_displayed(self):
+        return self.in_requests
+
+
+class RequestDetailsToolBar(RequestView):
+    copy = Button(title='Copy original Request')
+    edit = Button(title='Edit the original Request')
+    delete = Button(title='Delete this Request')
+    reload = Button(title='Reload the current display')
+    approve = Button(title='Approve this Request')
+    deny = Button(title='Deny this Request')
+
+
+class RequestDetailsView(RequestView):
+    @View.nested
+    class details(View):  # noqa
+        request_id = SummaryFormItem('Request Details', 'Request ID')
+        status = SummaryFormItem('Request Details', 'Status')
+        request_state = SummaryFormItem('Request Details', 'Request State')
+        requester = SummaryFormItem('Request Details', 'Requester')
+        request_type = SummaryFormItem('Request Details', 'Request Type')
+        description = SummaryFormItem('Request Details', 'Description')
+        last_message = SummaryFormItem('Request Details', 'Last Message')
+        created_on = SummaryFormItem('Request Details', 'Created On')
+        last_update = SummaryFormItem('Request Details', 'Last Update')
+        completed = SummaryFormItem('Request Details', 'Completed')
+        approval_state = SummaryFormItem('Request Details', 'Approval State')
+        approved_by = SummaryFormItem('Request Details', 'Approved/Denied by')
+        approved_on = SummaryFormItem('Request Details', 'Approved/Denied on')
+        reason = SummaryFormItem('Request Details', 'Reason')
+        provisioned_vms = SummaryFormItem('Request Details', 'Provisioned VMs')
+
+    @View.nested
+    class request(Tab):  # noqa
+        TAB_NAME = 'Request'
+        email = SummaryFormItem('Request Information', 'E-Mail')
+        first_name = SummaryFormItem('Request Information', 'First Name')
+        last_name = SummaryFormItem('Request Information', 'Last Name')
+        notes = SummaryFormItem('Request Information', 'Notes')
+        manager_name = SummaryFormItem('Manager', 'Name')
+
+    @View.nested
+    class purpose(Tab):  # noqa
+        TAB_NAME = 'Purpose'
+        apply_tags = BootstrapTreeview('all_tags_treebox')
+
+    @View.nested
+    class catalog(Tab):  # noqa
+        TAB_NAME = 'Catalog'
+        filter = SummaryFormItem('Select', 'Filter')
```

Comment:
I noticed you used several reserved python words in this PR. I don't think this is good idea.
filter, reload, etc shouldn't be used as names.